service/frontend/service.go

```diff
@@ -254,9 +254,7 @@ func (s *Service) Start() {
 	// Additional decorations
 	var handler Handler = s.handler
 	handler = NewDCRedirectionHandler(handler, s, s.config, s.params.DCRedirectionPolicy)
-	if s.params.Authorizer != nil {
-		handler = NewAccessControlledHandlerImpl(handler, s, s.params.Authorizer)
-	}
+	handler = NewAccessControlledHandlerImpl(handler, s, s.params.Authorizer, s.params.AuthorizationConfig)
```

Comment:
So it seems handler is being replaced straight away, we might need to remove either this line or the prev one?

Reply:
oh you may misunderstand. It's passing the old value and then replace to a new one.